Makefile

```diff
@@ -287,8 +290,23 @@ destroy-eks-cluster:
 .PHONY: deploy-test
 deploy-test:
 	@helm upgrade --install cloudzero-cloudwatch-metrics cloudzero/cloudzero-cloudwatch-metrics \
-        --namespace cloudzero-metrics --create-namespace \
+        --namespace $(namespace)-cloudzero-metrics --create-namespace \
         --set clusterName=$(namespace) \
         --set images.pullPolicy=Always \
         --set image.repository=$(image_repo) \
-        --set image.tag=$(image_tag)
\ No newline at end of file
+        --set image.tag=$(image_tag)
+
+
+.PHONY: init
+init:
+	@go_version=$$(go version | cut -d' ' -f3 | sed 's/go//'); \
+	check_string=$(REQUIRED_GO_VERSION); \
+	if (echo "$$go_version" | grep -q "$(REQUIRED_GO_VERSION)"); then \
```

Comment:
nit: I think this might return 0 if the version contains the correct version. for example, I think if the user somehow had a version of go that was 0.1.19, that would return as the correct version. I think that's super unlikely to happen, but you could probably make it less likely to happen it by adding the `-w` flag to the grep command